docs/src/firmware-action/get_started/02_coreboot_config.md

```diff
@@ -0,0 +1,12 @@
+## Create a coreboot configuration file
+
+Now we need to create a configuration file for coreboot.
+
+Either follow a [coreboot guide](https://doc.coreboot.org/tutorial/part1.html#step-5-configure-the-build) to get a base-bones-basic configuration, or just copy-paste this text into `seabios_defconfig` file.
```

Comment:
Either `bare-bones` or `base` but not `base-bones`

Reply:
ups :D